src/Components/Counter/Counter.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export class Counter extends React.Component {
```

Comment:
In `App.js` defaut export is used for the `App` component.
Here `Counter` uses named export.
Let's use one approach in our component. It doesn't really matter what export you will choose.